HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -1537,6 +1537,28 @@ def self.apply_hvac_control(hpxml, schedules_file)
         hvac_control.cooling_setup_start_hour_isdefaulted = true
       end
 
+      if hvac_control.onoff_thermostat_deadband.nil?
+        hvac_control.onoff_thermostat_deadband = 0.0
+        hvac_control.onoff_thermostat_deadband_isdefaulted = true
+      end
+
+      # Do not apply on off thermostat if timestep is >= 2
+      # Only availabe with 1 min time step
```

Comment:
This is because our EMS program to ramp up capacity happens within 2 mins, if the time step is over 1 min, then the impact is not captured.